action.yml

```diff
@@ -1,38 +1,26 @@
 name: 'Add Navbar Action'
-description: 'Adds a top navigation bar to Documenter.jl-generated sites in the gh-pages branch.'
+description: 'Adds a top navigation bar to Documenter.jl-generated sites'
 inputs:
+  doc-path:
+    description: 'Path to the Documenter.jl output'
+    required: false
+    # Most julia projects have it here
+    default: 'docs/build'
   navbar-url:
     description: 'URL of the navbar HTML to be inserted.'
-    required: true
+    required: false
+    default: 'https://raw.githubusercontent.com/TuringLang/turinglang.github.io/main/assets/scripts/TuringNavbar.html'
   exclude-paths:
     description: 'Comma-separated list of paths to exclude from navbar insertion.'
     required: false
     default: ''
-  token:
-    description: 'GitHub token for authentication'
-    required: true
+
 runs:
   using: "composite"
   steps:
-    - name: Checkout gh-pages
-      uses: actions/checkout@v4
-      with:
-        ref: gh-pages
-        fetch-depth: 0
-    - name: Configure Git
-      run: |
-        git config user.name github-actions[bot]
-        git config user.email github-actions[bot]@users.noreply.github.com
-      shell: bash
     - name: Update Navbar
+      working-directory: ${{ inputs.doc-path }}
       run: |
         chmod +x ${{ github.action_path }}/scripts/insert_navbar.sh
         ${{ github.action_path }}/scripts/insert_navbar.sh . "${{ inputs.navbar-url }}" --exclude "${{ inputs.exclude-paths }}"
-        if [[ -n $(git status -s) ]]; then
-          git add .
-          git commit -m "Added navbar using insert_navbar.sh"
-          git push "https://${{ github.actor }}:${{ inputs.token }}@github.com/${{ github.repository }}.git" gh-pages
-        else
-          echo "No changes to commit"
-        fi
-      shell: bash
```

Comment:
We no longer need to git commit / push here as Documenter's deploy will sort that for us.

Reply:
Ohk!